react-native-purchases-ui/ios/PaywallViewWrapper.m

```diff
@@ -58,4 +58,18 @@ - (void)layoutSubviews {
     }
 }
 
+- (void)setOptions:(NSDictionary *)options {
+    if (@available(iOS 15.0, *)) {
+        NSDictionary *offering = options[@"offering"];
+        if (offering && ![offering isKindOfClass:[NSNull class]]) {
+            NSString *identifier = offering[@"identifier"];
+            if (identifier) {
+                [(RCPaywallViewController *)self.paywallViewController updateWithOfferingIdentifier:identifier];
```

Comment:
We don't need to cast this anymore

Reply:
hmmm isn't `self.paywallViewController` a `UIViewController`?

Reply:
But we can change the type now that PaywallProxy returns `PaywallViewController`.